README.md

```diff
@@ -7,13 +7,9 @@
 
 Read the [full documentation](https://slackapi.github.io/node-slack-sdk) for all the lovely details.
 
-So you want to build a Slack app with Node.js? We've got you covered. `node-slack-sdk` is aimed at making
-building Slack apps ridiculously easy. This module will help you build on all aspects of the Slack platform,
-from dropping notifications in channels to fully interactive bots.
+This library is a wrapper around the Slack [RTM](https://api.slack.com/rtm) and [Web](https://api.slack.com/web) APIs. 
 
-This is a wrapper around the Slack [RTM](https://api.slack.com/rtm) and [Web](https://api.slack.com/web) APIs.
-
-This library provides the low level functionality you need to build reliable apps and projects on top of Slack's APIs.
+It will help you build on all aspects of the Slack platform, from dropping notifications in channels to fully interactive bots. It provides the low level functionality you need to build reliable apps and projects on top of Slack's APIs.
```

Comment:
ehh, i don't agree with this. this module covers the Web API, RTM API (which you mention above) and Incoming Webhooks. it does not cover Events API, Interactive Messages, Outgoing Webhooks, or Slash Commands. want to try again?